docs/docs/userguide/CONFIGURATION.md

```diff
@@ -157,3 +157,43 @@ db_allocated_storage = 100
 ```terraform
 db_iops = 1000
 ```
+
+### Bamboo License
```

Comment:
I think somewhere we should create a `Sensitive Data` section/warning and link https://www.terraform.io/docs/language/state/sensitive-data.html and https://kubernetes.io/docs/concepts/configuration/secret/#security-properties so users can read about implications.

I find `Kubernetes Secrets` the most misleading naming in the whole ecosystem.